src/plugins/intel_cpu/tests/functional/behavior/import_model_fuzzy.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <string>
+#include <sstream>
+#include <fstream>
+
+#include "openvino/openvino.hpp"
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "openvino/pass/serialize.hpp"
+
+TEST(CPUImportModel, Fuzzy) {
+    auto type = ov::element::f32;
+    auto input = std::make_shared<ov::op::v0::Parameter>(type, ov::Shape{16, 8});
+    auto tensor = ov::test::utils::create_and_fill_tensor(type, ov::Shape{8, 16});
+    auto weight = std::make_shared<ov::op::v0::Constant>(tensor);
+    auto matmul = std::make_shared<ov::op::v0::MatMul>(input, weight);
+    auto model = std::make_shared<ov::Model>(matmul, ov::ParameterVector{input}, "import_model");
+    ov::Core core;
+    auto compiled_model = core.compile_model(model, "CPU");
+    // Import model with compiled model works fine.
```

Comment:
Maybe instead add comment make two test case with description for correct and incorrect model importing?